src/main/java/com/online/bookstore/repository/BookRepository.java

```diff
@@ -1,7 +1,27 @@
 package com.online.bookstore.repository;
 
 import com.online.bookstore.model.Book;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.EntityGraph;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface BookRepository extends JpaRepository<Book, Long> {
+
```

Comment:
do not start class/interface with empty line (fix everywhere)

Reply:
rewritten